pkg/config/constants/constants.go

```diff
@@ -125,4 +125,7 @@ const (
 	LocalPathStorageNamespace string = "local-path-storage"
 
 	TestVMLabel = "istio.io/test-vm"
+
+	// TrustworthyJWTPath is the defaut 3P token to authenticate with google services
```

Comment:
update the comments, why only google services

Reply:
Typo. Resolved this.